src/ralph/admin/widgets.py

```diff
@@ -128,12 +128,22 @@ def render_options(self, choices, selected_choices):
 
 class AutocompleteWidget(forms.TextInput):
     def __init__(self, field, admin_site, attrs=None, using=None, **kwargs):
+        self.many_to_many = kwargs.get('many_to_many', False)
```

Comment:
wouldn't it be better to create separated widget for m2m (maybe subclassing `AutocompleteWidget`)? @ar4s ?